src/tests/functional/shared_test_classes/src/single_op/inverse.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_op/inverse.hpp"
+
+#include "ov_models/builders.hpp"
+
+using namespace ov::test;
+
+namespace ov {
+namespace test {
+std::string InverseLayerTest::getTestCaseName(const testing::TestParamInfo<InverseTestParams>& obj) {
+    std::string test_type;
+    ov::Tensor input;
+    bool adjoint;
+    std::string device_name;
+
+    std::tie(test_type, input, adjoint, device_name) = obj.param;
+
+    const char separator = '_';
+    std::ostringstream result;
+    result << test_type << separator;
+    result << "input_shape=" << input.get_shape().to_string() << separator;
+    result << "type=" << input.get_element_type() << separator;
+    result << "adjoint=" << adjoint << separator;
+    result << "device=" << device_name;
+
+    return result.str();
+}
+
+void InverseLayerTest::SetUp() {
```

Comment:
This test does not check dynamic shapes scenario. It just create new graph for each new input. The right way is to pass a set of input shapes. Please look at the Random Uniform tests for example.